_benchmark/workloads/index.md

```diff
@@ -0,0 +1,249 @@
+
+### Operations
+
+The `operations` element lists the API OpenSearch operations performed by the workload For example, you can set an operation to `create-index`, which creates an index of Benchmarked data after the workload completes. Operations are usually listed inside of `schedule`.
+
+### Schedule
+
+The `schedule` element contains a list of actions and  operations run by the workload. Operations run according to the order in which they appear in the `schedule`. The following example illustrates a `schedule` with multiple operations, each defined by their `operation-type`. 
+
+```json
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "green"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": 5000
+      },
+      "warmup-time-period": 120,
+      "clients": 8
+    },
+    {
+      "operation": {
+        "name": "query-match-all",
+        "operation-type": "search",
+        "body": {
+          "query": {
+            "match_all": {}
+          }
+        }
+      },
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+According to this schedule, each action will run in the following order.
+
+1. The `create-index` operation creates an index with benchmarked data.
+2. The `cluster-health` operation assess the health of the cluster before running the workload. In this example, the workload will wait until the status of cluster's health is `green`.
+3. The `bulk` operation run the `bulk` API to index `500` documents at once.
+4. Before benchmarking, the workload waits until the specified `warmup-time-period` passes. In this example, the warmup period if `120` seconds.
+5. The `clients` options defines the number of clients that will run the remaining actions in the schedule concurrently.
+6. The `search` runs a `match_all` query to find documents that match each other after they are indexed by the `bulk` API using the 8 clients specified.
+7. The `iterations` options indicates the number of time each client runs the `search` operation. The report generated by the benchmark automatically adjusts the percentile numbers based on this number. To generate a precise percentile, the benchmark needs to run at least 1000 iteration.
+8. Lastly, the `target-throughput` defines the number of requests per second each client performs, which when set can help reduce the latency of the benchmark. For example, a `target-throughput` of 100 requests divided by 8 clients means that each client will issue 12 requests per second.
+
+
+## Workload examples
+
+If you want to try certain workloads before creating your own, use the following examples.
+
+### Running unthrottled
```

Comment:
`Running unthrottled` and `Workload with a single task` being under `Workload examples` is a little misleading.  `Running unthrottled` and `Workload with a single task` are examples of somewhat niche use-cases and aren't basic examples of what users might have just learned. 

Maybe we can move these two to a section beneath it called "Tailoring Workloads to Specific Use Cases" or something more fitting and then add a super small example workload that they can use in "Workload Example"